src/models/Settings.php

```diff
@@ -93,6 +96,16 @@ class Settings extends Model
      */
     public $minimumTotalPriceStrategy = 'default';
 
+    /**
+     * @var string
+     */
+    public $freeOrderPaymentStrategy = 'complete';
+
+    /**
+     * @var bool
+     */
+    public $mergeLastCartOnLogin = true;
```

Comment:
Merged thsi PR, but `mergeLastCartOnLogin` is an old config setting from commerce 2 which I removed.

Thanks.